File: JSBASIC/89_perfect_square.js
// 89	89_perfect_square.js	Check if a number is a perfect square using Math.sqrt().

let num = 49;
let sqrt = Math.sqrt(num);
console.log("Number:", num);
if (sqrt === Math.floor(sqrt)) {
    console.log(num, "is a Perfect Square. Square root =", sqrt);
} else {
    console.log(num, "is NOT a Perfect Square.");
}

File: JavaBasic/89_perfect_square.java
// 89	89_perfect_square.java	Check if a number is a perfect square using Math.sqrt().
public class _89_perfect_square {
    public static void main(String[] args) {
        int num = 49;
        double sqrt = Math.sqrt(num);
        System.out.println("Number: " + num);
        if (sqrt == (int)sqrt) System.out.println(num + " is a Perfect Square. sqrt = " + (int)sqrt);
        else System.out.println(num + " is NOT a Perfect Square.");
    }
}
